pandas/core/indexes/multi.py

```diff
@@ -2328,23 +2337,40 @@ def convert_indexer(start, stop, step, indexer=indexer, labels=labels):
             j = labels.searchsorted(loc, side='right')
             return slice(i, j)
 
-    def get_locs(self, tup):
+    def get_locs(self, seq):
         """
-        Given a tuple of slices/lists/labels/boolean indexer to a level-wise
-        spec produce an indexer to extract those locations
+        Get location for a given label/slice/list/mask or a sequence of such as
+        a integer, array of integers, slice or boolean mask.
 
         Parameters
         ----------
-        key : tuple of (slices/list/labels)
+        seq : label/slice/list/mask or a sequence of such.
 
         Returns
         -------
-        locs : integer list of locations or boolean indexer suitable
-               for passing to iloc
+        locs : array of integers, slice or boolean mask suitable for passing to
+               iloc
+
+        Examples
+        ---------
+        >>> mi = pd.MultiIndex.from_arrays([list('abb'), list('def')])
+
+        >>> mi.get_locs('b')
+        slice(1, 3, None)
```

Comment:
If I run this locally, I actually get an array as result:

```
In [109]: mi = pd.MultiIndex.from_arrays([list('abb'), list('def')])

In [110]: mi.get_locs('b')
Out[110]: array([1, 2])

```

Reply:
Yeah, me too, actually. It seems the func then only return arrays?